src/Pyz/Yves/SspDashboardManagement/Theme/default/views/dashboard/dashboard.twig

```diff
@@ -2,7 +2,7 @@
 
 {% block customerContent %}
     {% block userInfo %}
-        <div>
+        <div data-qa="welcome-block">
```

Comment:
We must be able to remove this project-level override, core has this part fixed already

Reply:
Same for the `information` block. Please try to remove the override to reduce the complexity on the project's side.

Reply:
Core and project have different html structure. @supproduction could you check if it's possible to get rid of twig overriding?